app/capitalize_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  attr_accessor :person
+
+  def initialize(person)
+    super()
```

Comment:
- [ ] [OPTIONAL] Kindly remove `super( )` in the `initialize` method inside `CapitalizeDecorator` as it will cause an error when you try to create a new object of `CapitalizeDecorator`.